package.json

```diff
@@ -171,7 +166,11 @@
     "@emotion/jest": "11.3.0",
     "@loadable/babel-plugin": "5.13.2",
     "@loadable/webpack-plugin": "5.15.0",
+    "@storybook/addon-a11y": "6.3.1",
+    "@storybook/addon-controls": "6.3.1",
+    "@storybook/addon-docs": "6.3.1",
     "@storybook/addon-knobs": "6.2.9",
+    "@storybook/addon-viewport": "6.3.1",
     "@storybook/react": "6.2.9",
```

Comment:
Does this need to be changed to `6.3.1`? When I checkout the branch and run `yarn` and then `yarn run storybook`, I get a notification in the terminal saying `A new version (6.3.1) is available!`